gramps/gen/lib/eventtype.py

```diff
@@ -253,7 +256,8 @@ def _T_(value, context=''): # enable deferred translations
         ENGAGEMENT: _T_("engd.", "Engagement abbreviation"),
         DIVORCE: _T_("div.", "Divorce abbreviation"),
         DIV_FILING: _T_("div.f.", "Divorce Filing abbreviation"),
-        ANNULMENT: _T_("annul.", "Annulment abbreviation")
+        ANNULMENT: _T_("annul.", "Annulment abbreviation"),
+        STILLBIRTH: _T_("Stillbirth abbreviation|still.")
```

Comment:
Needs to use the new translation context:

`STILLBIRTH: _T_("still.", "Stillbirth abbreviation")`

Reply:
Fixed in [15a0085](https://github.com/gramps-project/gramps/pull/1148/commits/15a0085b6316bb277400a9133eabf32b47be2558)